pettingzoo/utils/env.py

```diff
@@ -86,10 +86,11 @@ def observe(self, agent: str) -> Optional[ObsType]:
         """
```

Comment:
Just leaving this comment here that maybe `self.render_mode` should be added into this superclass.